src/aznfswatchdog

```diff
@@ -65,33 +65,60 @@ chattr -f +i $RANDBYTES
 # kernels. If we find an entry stuck for more than a certain time in SYN_SENT state
 # we delete the entry so that kernel looks up fresh NAT rules and creates a new entry.
 #
-declare -A cthash_synsent
-declare -A cthash_synsent_now
+declare -A cthash_stuck
+declare -A cthash_stuck_now
 
 reconcile_conntrack_one()
```

Comment:
rename this to reconcile_conntrack_synsent and add a new one reconcile_conntrack_unreplied